tests/remote/integration_google_cloud/variables.tf

```
@@ -0,0 +1,19 @@
+variable "client_email" {
+  type = string
+}
+
+variable "private_key" {
+  type = string
```

```suggestion
  type = string
  sensitive = true
```
Related to comment in `outputs.tf`

---

Applied!